src/sdk/sql_cluster_router.cc

```diff
@@ -1828,40 +1829,22 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::HandleSQLCmd(const h
             exit(0);
         }
         case hybridse::node::kCmdShowJobs: {
```

Comment:
just in case. @lqy222 should testing the compatibility between different versions, like old SDK and newer OpenMLDB server. To insure old approach works as expect